src/coreclr/tools/aot/ILCompiler.Build.Tasks/ComputeManagedAssembliesToCompileToNative.cs

```diff
@@ -138,6 +138,7 @@ public override bool Execute()
                         }
                     }
 
+                    assembliesToSkipPublish.Add(taskItem);
```

Comment:
This one needed too. I miss that WindowsBase.dll then would be part of published application